### ISSUE TITLE
Stash-describing JSON uses floating point for timestamp

### ISSUE THREAD

**Issue by UnitedMarsupials** [sensu/uchiwa#541]
As transpired while investigating the #540, `timestamp`-field may (at least some times) be returned in _floating point_ format. For example:

``` json
  "timestamp":1.471097236e+09,
```

Given that the timestamp represents an _integer_ number of seconds since Unix Epoch (1471097236 in the above case, or Sat Aug 13 10:07:16 EDT 2016), why bother translating it into a floating point and back -- and _risk losing precision_ in the process?

Examining the port 3000 traffic with `tcpdump`, I see that the stash-request itself contains the timestamp as an integer number -- correctly:

``` json
{
 "content":{
  "reason":"Meow",
  "source":"uchiwa",
  "timestamp":1471099503
 },
 "dc":"som-sensu01",
 "path":"silence/som-sensu01",
 "expire":900
}
```

But, when that same request-description is coming back in response to the `/stashes`-query, the timestamp will be in floating point format... Incorrectly.

**Comment by palourde** [comment]
JSON Marshaling in Go automatically translates long number to floating number. Looks like there's an easy to keep the original representation: http://stackoverflow.com/a/22346593

**Comment by josephholsten** [comment]
This doesn't really help you, but you might be entertained that there is no such thing as an integer in JSON. JSON inheirits Javascript's numeric type, which is the IEEE 754 Double Precision floating point. Some interesting links on the point:
- http://yuiblog.com/blog/2009/03/10/when-you-cant-count-on-your-numbers/
- https://spacetelescope.github.io/understanding-json-schema/reference/numeric.html

**Comment by palourde** [comment]
Base on [this commit](https://github.com/golang/go/commit/92b3e3651dc44f54b458f171f641779f10fbaec0), it should be fixed within the Go langage in 1.8 release. From this point I will simply wait for this release.

**Comment by palourde** [comment]
I confirm that this issue has been automatically resolved with the 1.8 release of Go. Uchiwa 0.22.2 and newer are using this release.
